contribute.json

```diff
@@ -1,33 +1,32 @@
 {
-    "name": "Common Voice",
-    "description": "Common Voice is Mozilla's initiative to help teach machines how real people speak.",
-    "repository": {
-        "url": "https://github.com/mozilla/voice-web",
-        "license": "MPL2",
-        "tests": "https://travis-ci.org/mozilla/voice-web/"
-    },
-    "participate": {
-        "home": "https://voice.mozilla.org",
-        "docs":"https://github.com/mozilla/voice-web/blob/master/CONTRIBUTING.md",
-        "forum": "https://discourse.mozilla.org/c/voice",
-        "slack": "https://common-voice.slack.com/",
-        "slack-invite": "https://common-voice-slack-invite.herokuapp.com/"
```

Comment:
This file was re-indented by our formatting rules. These were the only lines actually removed.